<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QDialog" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="terminateButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>quit</string>
   </property>
  </widget>
  <widget class="QListView" name="ensembleDisplay">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>191</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ensembleLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dynamicLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="channelSelector">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>98</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="gainSlider">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>49</y>
     <width>24</width>
     <height>251</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QListView" name="programView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>131</y>
     <width>211</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="lnaState">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>108</width>
     <height>36</height>
    </rect>
   </property>
   <property name="value">
    <number>2</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="ifgainReduction">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>108</width>
     <height>36</height>
    </rect>
   </property>
   <property name="maximum">
    <number>59</number>
   </property>
   <property name="value">
    <number>20</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="autogainButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>94</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>autoGain</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
